spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/dynamodb/DynamoDbAutoConfigurationTest.java

```diff
@@ -263,6 +292,11 @@ public <T> TableSchema resolve(Class<T> clazz, String tableName) {
 			return null;
 		}
 
+		@Override
+		public <T> void register(TableSchema<T> tableSchema, String tableName) {
```

Comment:
Remove empty implementation once we have the default method in place.

Reply:
done